### DISCUSSION TITLE
QUDI module for optical spectrum analyzer

### DISCUSSION THREAD

**Discussion by Q-AUd** [Ulm-IQO/qudi-core#51] [Q&A] [answered]
Is there any QUDI module for optical spectrum analyzer? Specifically, I'm interested in Yokogawa AQ6370

**Comment by Neverhorst** [comment, marked as answer]
Hi @Q-AUd ,

not to my knowledge, no. At least not in the maintained repositories `qudi-core` and `qudi-iqo-modules`. Maybe someone out there wrote something that has not been pushed upstream. 

Any contribution regarding a qudi toolchain for spectrum analyzers would be highly appreciated, though... ;)
